app/mailers/alert_mailer.rb

```diff
@@ -19,4 +19,21 @@ def alert(alert, recipient)
     params[:reply_to] = @alert.reply_to unless @alert.reply_to.nil?
     mail(params)
   end
+
+  def self.send_msg_to_student(alert, recipient)
```

Comment:
I think adding the student and all the instructors to the recipients, rather than sending a separate email to each, is the best way to do this. That way, it can be the beginning of a conversation among the student, the instructors, and the Wiki Education staff member.

Reply:
Aaah. I have done a bit complicated code I guess. I will redo it, and will rebase since many things will disappear :)